File: src/Java/BehavioralPatterns/Command/FastFoodRestaurant/src/CommandFactory.java
import Commands.AddMenuItemCommand;
import Commands.ModifyMenuItemCommand;
import Commands.OrderCommand;
import Commands.RemoveMenuItemCommand;

public class CommandFactory {
    public OrderCommand getCommand(int commandOption) {
        switch (commandOption) {
            case 1:
                return new AddMenuItemCommand();
            case 2:
                return new ModifyMenuItemCommand();
            case 3:
                return new RemoveMenuItemCommand();
            default:
                return new AddMenuItemCommand();
        }
    }
}

File: src/PHP/BehavioralPatterns/Command/FastFoodRestaurant/CommandFactory.php
<?php

namespace BehavioralPatterns\Command\FastFoodRestaurant;

use BehavioralPatterns\Command\FastFoodRestaurant\Commands\AddMenuItemCommand;
use BehavioralPatterns\Command\FastFoodRestaurant\Commands\ModifyMenuItemCommand;
use BehavioralPatterns\Command\FastFoodRestaurant\Commands\OrderCommand;
use BehavioralPatterns\Command\FastFoodRestaurant\Commands\RemoveMenuItemCommand;

class CommandFactory {
    public function getCommand(int $commandOption): OrderCommand {
        return match ($commandOption) {
            2 => new ModifyMenuItemCommand(),
            3 => new RemoveMenuItemCommand(),
            default => new AddMenuItemCommand(),
        };
    }
}